CMakeLists.txt

```diff
@@ -174,6 +174,8 @@ else()
   message( FATAL_ERROR "No WASM compiler cound be found" )
 endif()
 
+FIND_PACKAGE(binaryen REQUIRED)
```

Comment:
> this creates a code link to binaryen's 1.37.x

Is there a way we can enforce that somehow here? I know that will require changes to the FindBinaryen.cmake

Reply:
sadly no, there seems to be no header level or file level build stamping.   If we want to assume that binaryen's directory is a git repro we could look at the tag, however our standard install script does _not_ survive the git repo